### PR TITLE
Gun stuff. Some fixes, some changes, and a shiny new SVD

### PR DESCRIPTION
**Pull request by SpadesNeil**
- **Fixes** the heavy sniper hand sprite so it no longer looks like a pistol. Not ideal, but better.
- **Fixes** descriptions lacking what rounds a gun uses.
- **Fixes** sprites for shotgun ammo clips.
- **Fixes** energy sniper back sprite not appearing.
- **Adds** the SVD back by request of Woodrat. It's not spawnable yet, since I don't know where you want to balance it. Imagine the Z8 rifle, but except instead of a grenade launcher, it gets a scope.
- **Changes** energy sniper to use the Z8's hand sprite, because it's more distinct than looking the same as the laser carbine.
- **Changes** some descriptions to have better grammar.
- **Changes** some descriptions to have more interesting fluff.
- **Changes** AS-24 auto shotgun description so it's no longer copied almost word for word from the assault rifle.
- **Changes** Desert Eagle icons to no longer look like ass.
Before:
deagle-old
After (from /tg/ station):
deagle

### CODE REVIEW

**Comment by Anewbe** [comment]
Did you test all of this, thoroughly?

**Comment by SpadesNeil** [comment]
Yes. All of this was tested. I was very bored that day.

File: code/modules/projectiles/guns/projectile/sniper.dm [hunk 1]
```diff
@@ -4,8 +4,7 @@
 	name = "anti-materiel rifle"
 	desc = "A portable anti-armour rifle fitted with a scope, the HI PTR-7 Rifle was originally designed to used against armoured exosuits. It is capable of punching through windows and non-reinforced walls with ease. Fires armor piercing 14.5mm shells."
 	icon_state = "heavysniper"
-	item_state = "l6closed-empty" // placeholder
-	item_state_slots = list(slot_r_hand_str = "heavysniper", slot_l_hand_str = "heavysniper")
+	item_state_slots = list(slot_r_hand_str = "l6closed-empty", slot_l_hand_str = "l6closed-empty") // placeholder
```
Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/sniper.dm]: This doesn't look great but at least it doesn't look like a pistol anymore.

**Comment by SpadesNeil** [comment]
Any word on this yet?

**Pull request merged**